client/me/help/help-contact/index.jsx

```diff
@@ -64,6 +65,10 @@ const HelpContact = React.createClass( {
 			this.props.connectHappychat();
 		}
 
+		if ( config.isEnabled( 'help/directly' ) ) {
+			this.props.initializeDirectly();
```

Comment:
I haven't, I've mostly followed existing patterns in this component — would you mind explaining a little further? (I'm still very new to Calypso).

I see there's [AsyncLoad](https://github.com/Automattic/wp-calypso/tree/master/client/components/async-load) which seems to be for loading components. I'm not 100% sure how to use that here, unless the suggestion is to encapsulate all the Directly stuff in its own component?